controllers/machineset_controller.go

```diff
@@ -173,10 +173,6 @@ func (r *MachineSetReconciler) reconcile(ctx context.Context, cluster *clusterv1
 		}
 	}
 
-	// Make sure selector and template to be in the same cluster.
-	machineSet.Spec.Selector.MatchLabels[clusterv1.ClusterLabelName] = machineSet.Spec.ClusterName
-	machineSet.Spec.Template.Labels[clusterv1.ClusterLabelName] = machineSet.Spec.ClusterName
-
```

Comment:
I believe we should be leaving the `ClusterLabelName` here for now, and removing the ones below that are adding `MachineSetLabelName` below.

Reply:
Sorry! yeah i deleted the wrong lines

Reply:
Not a problem at all. Thanks for the quick fix :)